source/_data/projects.yaml

```diff
@@ -0,0 +1,49 @@
+parameters:
+    skills:
+        - 'PHP'
+        - 'HTML'
+        - 'CSS'
+        - 'Javascript'
+        - 'LESS'
+        - 'SCSS'
+        - 'Python'
+        - 'C'
+        - 'C++'
+        - 'MySQL'
+        - 'MongoDB'
+        - 'SQLite'
+        - 'Nette Framework'
+        - 'Symfony'
+        - 'Silex'
+        - 'Slim'
+        - 'PHPCake'
+        - 'NodeJS'
+        - 'ReactJS'
+        - 'VueJS'
+        - 'PhpUnit'
+        - 'jQuery'
+        - 'Composer'
+        - 'Mootools'
+        - 'Doctrine'
+        - 'RabbitMQ'
+        - 'Firebase'
+        - 'Docker'
+        - 'Xdebug'
+        - 'Blackfire'
+        - 'SEO'
+        - 'Photoshop'
+        - 'Gulp'
+        - 'Grunt'
+        - 'Webpack'
+        - 'Blockchain'
+        - 'Smart Contracts (Solidity)'
+        - 'Automatized Trading Systems (Easy Language)'
+        - 'GIT'
+        - 'SVN'
+        - 'Linux server administrace'
+        - 'Continuous Integration'
+        - 'Continuous Delivery'
+        - 'Continuous Deployment'
+        - 'DevOps'
+        - 'Clean code'
+        - 'Solid principy'
```

Comment:
This is to show how parameter works.

It's good practise to decouple all *many-lists* to parameters to keep templates small